modules/delegates.js

```diff
@@ -272,25 +281,30 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
+	var secretsList;
 
 	if (library.config.forging.secret) {
 		if (Array.isArray(library.config.forging.secret)) {
-			secrets = library.config.forging.secret;
+			secretsList = library.config.forging.secret;
 		} else {
-			secrets = [library.config.forging.secret];
+			secretsList = [library.config.forging.secret];
 		}
 	}
 
-	if (!secrets || !secrets.length) {
+	if (!secretsList || !secretsList.length || !library.config.forging.force) {
 		return setImmediate(cb);
 	} else {
-		library.logger.info(['Loading', secrets.length, 'delegates from config'].join(' '));
+		library.logger.info(['Loading', secretsList.length, 'delegates from config'].join(' '));
 	}
 
-	async.eachSeries(secrets, function (secret, cb) {
+	async.eachSeries(secretsList, function (encryptedItem, cb) {
+		var secret = __private.decryptSecret(encryptedItem.encryptedSecret, library.config.forging.defaultKey);
```

Comment:
Why we won't be able to test correctly?
If secret in config is not encrypted - delegate will start forging at node start, if it's encrypted - it can require a start by API and pass needs to be provided - we can trigger it from tests. We also can put in config 101 with plain secrets and put some more delegates or (even ones duplicated from 101) and then disable and enable forging to test it.
It's much more nice to deal with plain secrets during play with local testnets.